pkgcore/operations/format.py

```diff
@@ -45,10 +45,11 @@ def fetch_one(self, fetchable, observer):
         # XXX hack atm, could use better logic but works for now
         try:
             fp = self.fetcher(fetchable)
-        except Exception:
+        except Exception as e:
+            observer.error(str(e))
+            observer.error("failed fetching %s", fetchable)
             fp = None
         if fp is None:
-            self.failed_fetch(fetchable, observer)
```

Comment:
Yeah, that's what I meant :). Seems like there only two cases, one is:
NEWS.rst:162:- Run pkg_nofetch phase when any files in SRC_URI fail to be fetched.
and another one if you want to run pebuild <atom> nofetch,  right ?